app/components/Layout.tsx

```diff
@@ -44,6 +44,15 @@ export function Layout({ back, children, title }: LayoutProps) {
 						Code of Conduct
 					</InternalAnchor>
 				</div>
+				<div className={styles.footerLinks}>
+					<InternalAnchor reloadDocument to="/events" variant="heavy">
```

Comment:
(Digging through code, I found this: https://github.com/SquiggleTools/boston-ts-website/blob/main/app/components/Anchor.tsx#L8-L31, is it intentional that `InternalAnchor` uses `ExternalAnchorProps` and `ExternalAnchor` uses `InternalAnchorProps`?
EDIT: I think it's just a typo. Happy to open my first PR if you'd like 😂)

Reply:
Haha nice catch! Yeah that is a typo, good one. +1 to your first PR 😄

Reply:
Opened at https://github.com/SquiggleTools/boston-ts-website/pull/287